flow/connectors/mysql/logger.go

```diff
@@ -0,0 +1,117 @@
+package connmysql
```

Comment:
I'm not sure why this is needed

Reply:
binsyncer in go-mysql uses https://github.com/siddontang/go-log